packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -180,12 +180,30 @@ class DocumentCollection {
     }
   }
 
-  async update(document) {
-    const resp = await this.stackClient.fetchJSON(
-      'PUT',
-      uri`/data/${this.doctype}/${document._id}`,
-      document
-    )
+  async update(document, options = {}) {
+    const { retriesOnConflict = 0 } = options
+    let resp
+    try {
+      resp = await this.stackClient.fetchJSON(
+        'PUT',
+        uri`/data/${this.doctype}/${document._id}`,
+        document
+      )
+    } catch (error) {
+      const { status } = error
+      if (status === 409 && retriesOnConflict) {
```

Comment:
```suggestion
      if (status === 409 && retriesOnConflict && retriesOnConflict > 0) {
```

Reply:
A number less or equal than 0 isn't considered as a falsy value?

Reply:
`if (-1) { console.log("true") } else { console.log("false") }`
--> "true"

Reply:
OK I was wrong, thanks 👍

Reply:
Only `0` is falsy.